src/Cache/_ScsDataClient.php

```diff
@@ -224,4 +249,119 @@ public function listPushBack(
         return new CacheListPushBackResponseSuccess();
     }
 
+    public function listPopFront(string $cacheName, string $listName) : CacheListPopFrontResponse
+    {
+        try {
+            validateCacheName($cacheName);
+            validateListName($listName);
```

Comment:
Don't we want to catch `InvalidArgumentError` for those?

Reply:
We already do! It's an `SdkError` subclass which is handled in the first `catch` below.